### PR TITLE
Add github action to publish to pypi

### PR DESCRIPTION
**Pull request by forslund**
#### Description
Adds github action support for automatically publishing new releases to pypi.Requires PYPI_PASSWORD to be added to the secrets to work.

Edit: (12/8) Updated to match the flow in adapt.

#### Type of PR
- [ ] Bugfix
- [ ] Feature implementation
- [ ] Refactor of code (without functional changes)
- [ ] Documentation improvements
- [ ] Test improvements
- [x] CI/CD Improvement

### CODE REVIEW

**Comment by krisgesling** [comment]
Just noting that we'll validate this on Adapt first and then replicate once it's all confirmed working.

**Comment by forslund** [comment]
Good plan

**Comment by forslund** [comment]
This is now updated to match the flow in adapt

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ release/* ]
```
Comment (krisgesling) [inline .github/workflows/push.yml:9]: We took this out of Adapt, but really it just allows us to manually tag releases right?

**Review by krisgesling — APPROVED** [review]
Have added an API token for PYPI_PASSWORD so this should be good to go :)

File: .github/workflows/push.yml (hunk 1 above)
Comment (forslund) [reply .github/workflows/push.yml:9]: I'll remove it

**Comment by forslund** [comment]
Removed the on release tag criteria.

**Review by krisgesling — APPROVED** [review]

**Pull request merged**